src/fragments/lib/auth/common/mfa/flows.mdx

```diff
@@ -0,0 +1,318 @@
+
+### Recovering from a lost TOTP device
+
+If a user loses access to their TOTP device, they would need to contact an administrator to help get access to their account. Based on the Cognito User Pool configuration, the administrator can use the [AdminSetUserMFAPreference](https://docs.aws.amazon.com/cognito-user-identity-pools/latest/APIReference/API_AdminSetUserMFAPreference.html) to either change the MFA preference to a different MFA method or to disable MFA for the user.
+
+<Callout warning>
+
+In a scenario where MFA is marked as **Required** in Cognito User Pool and another MFA method is not set up, the administrator would need to first initiate an [AdminUpdateUserAttributes](https://docs.aws.amazon.com/cognito-user-identity-pools/latest/APIReference/API_AdminUpdateUserAttributes.html) call and update the user’s phone number attribute. Once this is complete, the administrator can continue changing the MFA preference to SMS as suggested above.
+
+</Callout>
+
+
+## MFA Preferences
+
+You can use Time-based One-Time Password (TOTP) for multi-factor authentication (MFA) in your web or mobile applications. Amplify's Auth category includes support for handling TOTP setup and codes, offering an integrated solution for enhanced security for your users. Authenticator apps generate the temporary, one-time passwords. These apps, such as Google Authenticator, Microsoft Authenticator, have the TOTP algorithm built-in and work by using a shared secret key and the current time to generate a short-lived password.
+
+<Callout>
+The APIs for fetching and updating MFA preferences are only available on the AWS Cognito Auth plugin and not on the Auth category. 
+</Callout>
+
+### Fetch the current user's MFA preferences
+
+Invoke the following API to get the MFA preference for the current user.
+
+import iosFetchMFAPreference from "/src/fragments/lib/auth/ios/mfa_preference/10_fetch_mfa_preference.mdx";
+
+<Fragments fragments={{ios: iosFetchMFAPreference}} />
+
+### Update the current user's MFA preferences
+
+Invoke the following API to update the MFA preference for the current user.
+
+<Callout warning>
+
+Only one MFA method can be marked as preferred at a time. If the user has multiple MFA methods enabled and tries to mark more than one MFA method as preferred, the API will throw an error.
+
+</Callout>
+
+import iosUpdateMFAPreferences from "/src/fragments/lib/auth/ios/mfa_preference/20_update_mfa_preference.mdx";
+
+<Fragments fragments={{ios: iosUpdateMFAPreferences}} />
+
+If multiple MFA methods are enabled for the user, the `signIn` API will return `continueSignInWithMFASelection` as the next step in the auth flow. During this scenario, the user should be prompted to select the MFA method they want to use to sign in.
+
+Refer to the [Continue signin with MFA Selection](/lib/auth/signin_next_steps#continue-signin-with-mfa-selection) section for more details.
+
+<Callout>
+
+<b>AWS Cognito Limitation</b>
```

Comment:
For this callout - I think this should just be a part of the SMS MFA and TOTP setup flows - that you have to call the set MFA preference after set up so that you can then retrieve this information when you call the getMFAPreferences API

Reply:
Done. Moved call outs to both SMS and TOTP sections.

Reply:
@dnys1 This was the reason I moved the callouts in the other sections rather than preferences.